plugins/http_plugin/http_plugin.cpp

```diff
@@ -133,7 +133,10 @@ namespace eosio {
       struct abstract_conn {
          virtual ~abstract_conn() {}
          virtual bool verify_max_bytes_in_flight() = 0;
+         virtual bool verify_max_requests_in_flight() = 0;
          virtual void handle_exception() = 0;
+
+         virtual void update_connection(const std::string & body, int code) = 0;
```

Comment:
I think a more appropriate name for this method would be `send_response` 

As the responses can be quite large and the back-end supports it we do want to `std::move` the body.  @heifner is correct that the `std::move` does nothing for a `const std::string&` however, I think the fix went the wrong direction.

Instead, this method should take a `std::string` (no reference) and then `std::move` from it. 

The caller is already `std::move`'ing into this argument so, that should move it all the way from the caller to the backend without creating a copy.

Reply:
> I think a more appropriate name for this method would be `send_response`
> 
> As the responses can be quite large and the back-end supports it we do want to `std::move` the body. @heifner is correct that the `std::move` does nothing for a `const std::string&` however, I think the fix went the wrong direction.
> 
> Instead, this method should take a `std::string` (no reference) and then `std::move` from it.
> 
> The caller is already `std::move`'ing into this argument so, that should move it all the way from the caller to the backend without creating a copy.

Changed the function.